src/style.css

```diff
@@ -47,5 +66,8 @@ input {
 }
 
 .submit {
-  width: 50%;
+  /* width: 50%; */
```

Comment:
[OPTIONAL] Please don't leave any commented code in your codebase. In this way, you can keep your codebase clean, organized and professional.